models/balancer/gnosis/balancer_v2_gnosis_pools_fees.sql

```diff
@@ -0,0 +1,43 @@
+{{
+    config(
+        schema = 'balancer_v2_gnosis',
+        alias= 'pools_fees',
+        materialized = 'incremental',
+        file_format = 'delta',
+        incremental_strategy = 'merge',
+        unique_key = ['block_number', 'tx_hash', 'index'],
+        post_hook='{{ expose_spells(\'["gnosis"]\',
+                                    "project",
+                                    "balancer_v2",
+                                    \'["metacrypto", "jacektrocinski", "thetroyharris"]\') }}'
+    ) 
+}}
+
+{% set event_signature = '0xa9ba3ffe0b6c366b81232caab38605a0699ad5398d6cce76f91ee809e322dafc' %}
+{% set project_start_date = '2022-11-02' %}
+
+WITH registered_pools AS (
+    SELECT DISTINCT
+        `poolAddress` AS pool_address
```

Comment:
Maybe we can remove the `?
```suggestion
        poolAddress AS pool_address
```